hathorlib/utils.py

```diff
@@ -180,3 +183,15 @@ def get_hash160(public_key_bytes: bytes) -> bytes:
     h = hashlib.new('ripemd160')
     h.update(key_hash.digest())
     return h.digest()
+
+
+def is_nft_creation(tx: 'BaseTransaction') -> bool:
```

Comment:
Maybe `is_nft_token(tx: 'TokenCreationTransaction')`?

Reply:
When I created this method I thought about it being used by the tx-mining-service and full-node just like this`if is_nft_creation(tx):`

If I change to expect only token creation txs I would need to execute the conditional logic in both places.

I prefer receiving any base transaction and handle the conditional in the lib. What do you think?

Reply:
It would also make sense to make this a method of `BaseTransaction`, just like we have `is_block` or `is_transaction`. If it was `def is_nft_token(tx: 'TokenCreationTransaction') -> bool:` the implementation would be `return True` and `tx` wouldn't even mater.

Reply:
Oh, I just realized that we have a `is_nft_creation` method, but not on `BaseTransaction`. Maybe we could move that to `BaseTransaction` and use a structure like `is_block`/`is_transaction`.

Reply:
I really like your suggestion @jansegre. Did the refactor here https://github.com/HathorNetwork/python-hathorlib/pull/6/commits/25752a0ee6f891920efbaf6578f77991046d471d

Reply:
Looks great! Thanks!